src/resources/s3.js

```diff
@@ -446,6 +518,9 @@ class S3Bucket { // eslint-disable-line padded-blocks
 			if (sseParams) {
 				await this._putBucketEncryption(bucket.metadata.name, sseParams);
 			}
+			if (versioningConfiguration) {
+				await this._putVersioningConfiguration(bucket.metadata.name, versioningConfiguration);
+			}
```

Comment:
This `if` is correct, but for consistency we could just call `_putVersioningConfiguration` in any case, as it already is handling empty configurations correctly.